src/librbd/mirroring_watcher/Types.h

```diff
@@ -96,7 +96,4 @@ std::ostream &operator<<(std::ostream &out, const NotifyOp &op);
 } // namespace mirroring_watcher
 } // namespace librbd
 
-using librbd::mirroring_watcher::encode;
```

Comment:
Did you need to remove these for a reason? The ::encode/::decode should be strongly typed, so I would prefer to remain consistent w/ the rest of Ceph which just uses ```::encode(...)``` and ```::decode(...)```.